src/main/java/org/junit/internal/runners/model/EachTestNotifier.java

```diff
@@ -45,4 +45,22 @@ public void fireTestStarted() {
     public void fireTestIgnored() {
         notifier.fireTestIgnored(description);
     }
+
+    /**
+     * Notify {@link #notifier} that a test suite is about to start.
```

Comment:
It seems odd to use `@link` to link to a private field. How about this;

"Calls {@link RunNotifier#fireTestSuiteStarted(Description)}, passing the {@link Description} that was passed to the {@code EachTestNotifier} constructor. This
should be called when a test suite is about to be started."